orderer/common/cluster/block_fetcher.go

```diff
@@ -0,0 +1,104 @@
+package cluster
+
+import (
+	"github.com/hyperledger/fabric-protos-go/common"
+	"github.com/hyperledger/fabric/bccsp"
+)
+
+type suspectSet map[string]struct{}
+
+func (s *suspectSet) insert(entry string) {
+	if !s.has(entry) {
```

Comment:
you don't need the "if, since the value mapped is a struct{}, all struct{}s are identical. It doesn't matter if you override it.

Reply:
ok